autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/importing/ImportedPasswordValidator.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.impl.importing
+
+import com.duckduckgo.autofill.api.domain.app.LoginCredentials
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+interface ImportedPasswordValidator {
+    fun isValid(loginCredentials: LoginCredentials): Boolean
+}
+
+@ContributesBinding(AppScope::class)
+class DefaultImportedPasswordValidator @Inject constructor() : ImportedPasswordValidator {
+
+    override fun isValid(loginCredentials: LoginCredentials): Boolean {
```

Comment:
Just validating, do we think is correct when importing credentials that if no domain is present we don't import the entry? I'm checking and we allow users to add credentials with not domain, so wondering why we think they are not valid.

Reply:
Will take to Asana: [What to do if there is no domain in the imported credential?](https://app.asana.com/0/1208707347544984/1208707347544984/f)